.header.md

```diff
@@ -35,7 +35,7 @@ pool_configurations = {
 }
 ```
 
-The key of a `pool_config` is the name of the pool, following by its attributes, `ram_share_principals`, and a `sub_pools` map, which is another nested `pool_config`.
+The key of a `pool_config` variable is the name of the pool, followed by its attributes `ram_share_principals` and a `sub_pools` map, which is another nested `pool_config` variable.
```

Comment:
To avoid rebuilds caused by simple pool name changes, you can set the name of the pool also using a `name` variable which overrides the key's name value.

Reply:
Want me to add this line here?